src/app/components/common/common.component.ts

```diff
@@ -9,24 +10,37 @@ import { ContentAPIService } from '../../services/content.api.service';
 
 })
 
+@Injectable()
 export class CommonComponent implements OnInit {
 
-  protected stContent: string[] = [];
+  protected stContent: String[] = [];
   protected rtrnContent: String[][];
-  constructor(protected http: HttpClient, protected contentSrv: ContentAPIService, protected passedContent: string[]) {
-    this.stContent = passedContent;
-    console.log(passedContent);
+  constructor(protected http: HttpClient, 
```

Comment:
I don't think this does anything except for the ContentAPIService. So also not needed.

Reply:
we can eliminate